app/components/select_component/select_component.html.haml

```diff
@@ -0,0 +1,5 @@
+.select-label 

```

Comment:
this layout of label and input is repeated in all your components.
So create a generic component that has two slots; :label and :input
see FieldContainerComponent